report/report.go

```diff
@@ -47,6 +47,10 @@ var oopses = []*oops{
 				"BUG: unable to handle kernel paging request in %[1]v",
 			},
 			{
+				compile("BUG: unable to handle kernel paging request(?:.*\\n)+?.*IP: {{FUNC}}"),
```

Comment:
I would rather make the `{{PC}}` part optional in the previous regexp. There are too many of them already.